.github/workflows/workflow.yml

```diff
@@ -2,9 +2,9 @@ name: ci
 
 on:
   push:
-    branches: [ master, development, 'refactor/unit_tests**' ]
+    branches: [master, development, 'refactor/unit_tests**', 'main/**']
   pull_request:
-    branches: [ master, development, refactor/unit_tests ]
+    branches: [master, development, 'refactor/unit_tests**', 'main/**']
```

Comment:
I dont believe this is part of this PR.

Reply:
I am not sure why this is being shown as a difference coming from this PR. This change is already part of `development` and it was applied there using a different branch and a different PR.

Reply:
Since the contents in `workflow.yml` are the same we currently have in `development` I suggest we ignore that being shown by GitHub as changes.